examples/pytorch/rgcn/entity_sample_cugraph_relgraphconv.py

```diff
@@ -0,0 +1,179 @@
+import torch
```

Comment:
Add a concise description at the top of the file?
https://github.com/dmlc/dgl/blob/master/examples/sparse/sign.py

Reply:
Highlight the "what's new" and "key part that you want to sell the users" in this example in the description or in the code.

Reply:
Description added and the example now goes into the advanced directory.